spec/examples/util/initTestI18n.js

```diff
@@ -0,0 +1,34 @@
+import {init} from 'i18next';
+import formatFunc from '../../../src/util/i18nFormatting';
+
+// There is an issue when loading json directly that gives an error
+// saying it is not valid javascript.  I believe it may be due to
+// reusing the webpack config object, but I'm not sure.
+// https://github.com/webpack-contrib/json-loader/issues/13
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import enTestResourceFile from '!json-loader!./test-translation.json';
+
+export default function() {
+  // put the locale into the correct namespace for the i18n cache.
+  // Because we are adding it to the cache manually,
+  // we have to give it the hierarchy manually
```

Comment:
I realize this is quite heterodox but I generally like to avoid to inline comments in code—my general philosophy is that good code should be sufficiently self-documenting to not require additional comments. I think in this case you picked a good variable name and it’s pretty clear from that name what the intent is here!